### PR TITLE
Moved to classes and minor refactoring of data storage

### PR DESCRIPTION
**Pull request by mark-s**
### Why these changes?:
The `program.cs` class is a bit of a monster!
The logic contained in the warren of case blocks is difficult to read and will be tricky to maintain going forward.

### What's changed - in brief:
I've pulled out the code from each `if` branch into its own class. 
Used a collection of `Vulnerability` types rather than a `DataSet`.
Displays errors/exceptions instead of eating them.

### Why are these changes worth including?:
The codebase is now simpler and easier to understand and maintain; with each unit of functionality living in its own specific, focused class.

### What's changed - in detail:
As mentioned, the big change was pulling out the code from each case block in `program.cs` and moving each part into its own object.  **The code was not changed - only moved.**

The new Classes are found in the `/SuspectFiles` folder and are named after the system files that they `Check()`.

`PopulateVulnerabilities()` in `VulnerabilityCollection.cs` contains the details from the previous `DataSet` version.

`VulnerabilityCollection.cs` also builds a string containing any errors encountered during the checks, and will display these, along with the results, when `ShowResults()` is called.

I have also pulled out the info/logo code into a separate file `Info.cs` - just to clean things up a bit more. 

### What hasn't changed:
The version number.
The logic/code that was within the `case` blocks in the `program.cs` file

### CODE REVIEW

File: Watson/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -19,8 +18,6 @@
 // COM, set the ComVisible attribute to true on that type.
 [assembly: ComVisible(false)]
 
-// The following GUID is for the ID of the typelib if this project is exposed to COM
-[assembly: Guid("a678dceb-6a5f-4ae9-b122-deecefd6a87b")]
```
Comment (mark-s) [inline Watson/Properties/AssemblyInfo.cs:23]: This Guid isn't needed and can by used to 'fingerprint' the exe

File: Watson/VulnerabilityCollection.cs [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+                Console.WriteLine();
+            }
+
+            if (_vulnerabilities.Any(e => e.IsVulnerable))
+                Console.WriteLine(" [*] Finished. Found {0} vulns :)", _vulnerabilities.Count(i => i.IsVulnerable));
+            else
+                Console.WriteLine(" [*] Finished. Sorry, found 0 vulns :(");
+
+            // Output any errors found
+            Console.WriteLine(_errors);
```
Comment (mark-s) [inline Watson/VulnerabilityCollection.cs:41]: Delete this guy if writing the errors isn't wanted

**Comment by rasta-mouse** [comment]
Just goes to show how little I know about structuring an application.  Thanks for your contribution - it was very valuable and much appreciated.

**Comment by mark-s** [comment]
Happy to help!
You did the hard work, I just moved a few things around!

**Pull request merged**